src/styles/globals.css

```
@@ -10,6 +10,81 @@ body {
+  }
+
+  .settings-search-highlight {
+    border-radius: 0.25rem;
+    background: color-mix(
+      in srgb,
+      var(--color-secondary) 34%,
+      var(--color-text-base)
+    );
+
+    box-shadow: 0 0 0 1px
+      color-mix(in srgb, var(--color-secondary) 70%, transparent);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)